### ISSUE TITLE
WOFF2 seems to have replaced EOT for web use

### ISSUE THREAD

**Issue by Windsurfer01** [johanmattssonm/birdfont#164]
I am getting to grips with BirdFont, and it has many good features.
Unfortunately, after learning to create variable fonts, I found it does not export to WOFF2, which is essential for use with WordPress.
Would it be possible to provide a WOFF2 export in a later version?

**Comment by johanmattssonm** [comment]
Yes. I do want to add support for WOFF2. Maybe there is an online converter somewhere that you can use. WOFF2 is mainly a compression antilogarithm on top of regular TTF and OTF fonts so everthing should work as before after converting to WOFF2.

**Comment by Windsurfer01** [comment]
Thanks for the quick reply. I'll look for an EOT to WOFF2 converter.

**Comment by johanmattssonm** [comment]
Why EOT and not TTF/OTF? EOT was the good old format for Internet Explorer. I thought that I could remove support for that format.

**Comment by Windsurfer01** [comment]
I need a font that has certain characteristics and can be used both as WOFF2 in WordPress and as OTF or TTF in word processors. Being an open source advocate, I prefer OTF.

**Comment by Windsurfer01** [comment]
Sorry, I completely missed the point of your question! Of course, anything can be converted to WOFF2.
You know better than most people, that TTF is the best choice as the basis for a variable font, because it is based on quadratic Bézier curves, In the past I tended to use cubic Bézier curves for development, because I like to use the OTF format. 
Having developed a font in BirdFont with cubic curves, I was delighted to find that I could export it as TTF and then read it back to create a variable version.
